cli/src/pcluster/config/mappings.py

```diff
@@ -1111,6 +1113,7 @@
                 "default": "ondemand",
                 "allowed_values": ["ondemand", "spot"],
                 "cfn_param_mapping": "ClusterType",
+                "validators": [cluster_type_validator],
```

Comment:
This breaks with awsbatch where we can have `optimal`, instance families or a comma separated list of instances. shall we skip this validation with awsbatch?

Reply:
done